crates/bevy_winit/src/lib.rs

```diff
@@ -302,33 +300,32 @@ pub fn winit_runner(mut app: App) -> AppExit {
         SystemState::<CreateWindowParams<Added<Window>>>::from_world(app.world_mut());
     let mut winit_events = Vec::default();
     // set up the event loop
-    let event_handler = move |event, event_loop: &EventLoopWindowTarget<UserEvent>| {
-        let mut exit_status = handle_exit_status.lock().unwrap();
-
-        handle_winit_event(
-            &mut app,
-            &mut runner_state,
-            &mut create_window,
-            &mut event_writer_system_state,
-            &mut focused_windows_state,
-            &mut redraw_event_reader,
-            &mut winit_events,
-            &mut exit_status,
-            event,
-            event_loop,
-        );
-    };
+    let event_handler =
+        move |event: Event<RequestRedraw>, event_loop: &EventLoopWindowTarget<UserEvent>| {
```

Comment:
Can the types of these arguments be implied? I noticed in the old version `event` was implied, but `event_loop` was not.